bandit.yaml

```diff
@@ -0,0 +1,14 @@
+skips: [ B101, B404, B603 ]
+
+exclude_dirs:
+  - .eggs
+  - .git
+  - .pytest_cache
+  - .tox
+  - reference_files
+  - tests
+  - build
+  - compatibility_tests
```

Comment:
This directory doesn't exist for asdf-standard

Reply:
Its just yaml files so it should be ignored.